utils/src/main/kotlin/OrtProxySelector.kt

```diff
@@ -107,6 +108,16 @@ class OrtProxySelector(private val fallback: ProxySelector? = null) : ProxySelec
         proxiesForProtocol += proxy
     }
 
+    /**
+     * Add multiple [proxies for specific protocols][proxyMap] whose definitions come from [origin].
+     */
+    fun addProxyOrigin(origin: String, proxyMap: ProxyProtocolMap) =
```

Comment:
Maybe call this `addProxies`, because it is very similar to the function above, except it adds multiple proxies?

Reply:
Done.